jchuff.c

```diff
@@ -88,8 +106,8 @@ typedef struct {
 #else
 #if MAX_COMPS_IN_SCAN == 4
 #define ASSIGN_STATE(dest, src) \
-  ((dest).put_buffer = (src).put_buffer, \
-   (dest).put_bits = (src).put_bits, \
+  ((dest).put.buffer_simd = (src).put.buffer_simd, \
+   (dest).free_bits = (src).free_bits, \
```

Comment:
Why has put_bits been renamed to "free_bits"?  The comment explaining the purpose of that field has not changed.